src/OpenTelemetry.Instrumentation.Http/OpenTelemetryBuilderExtensions.cs

```diff
@@ -30,23 +30,19 @@ public static class OpenTelemetryBuilderExtensions
         /// </summary>
```

Comment:
The comment - can you edit it to say Enables HttpClient & HttpWebRequest instrumentation